launch.go

```diff
@@ -204,8 +210,33 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 		return fmt.Errorf("Fetching Pipeline ID %d: %v", j.PipelineID, err)
 	}
 
+	// If no parent build ID, get the parent event meta
 	if b.ParentBuildID == 0 {
-		log.Printf("This build has no Parent Build, so fetching Meta is skipped")
+		log.Printf("Fetching Parent Event %d", e.ParentEventID)
+		pe, err := api.EventFromID(e.ParentEventID)
+		if err != nil {
+			return fmt.Errorf("Fetching Parent Event ID %d: %v", e.ParentEventID, err)
+		}
+
+		// Write to "meta.json" file
+		metaFile := "meta.json"
```

Comment:
We can probably combine this and the code below in the `else`

Reply:
Refactored code to try to combine some overlapping logic
Lmk if that looks okay